web/web.go

```diff
@@ -169,50 +174,60 @@ func New(o *Options) *Handler {
 
 	instrh := prometheus.InstrumentHandler
 	instrf := prometheus.InstrumentHandlerFunc
+	ready := h.testReady
```

Comment:
In order to avoid confusion by reusing the name of the struct member and to keep consistency with the existing code, I'd call this `readyf`.